app/root.tsx

```diff
@@ -263,7 +263,6 @@ export function ErrorBoundary({ error }: Route.ErrorBoundaryProps) {
   useEffect(() => {
     if (errorDetails.shouldReport) {
       Sentry.captureException(error);
-      console.error(error, { time: new Date().toISOString() });
```

Comment:
Unrelated. I realized that react and react router already log this error so we were ending up with 3 of the same errors in the console. When I remove this there will be 2 but don't think I can tell react or react router not to log it